sdk/tables/azure-tables/review/tables.api.md

```diff
@@ -214,16 +214,16 @@ export interface Metrics {
     version?: string;
 }
 
+// @public
+export function odata(strings: TemplateStringsArray, ...values: unknown[]): string;
+
 // @public
 export type OdataMetadataFormat = "application/json;odata=nometadata" | "application/json;odata=minimalmetadata" | "application/json;odata=fullmetadata";
 
 // @public
-export interface QueryOptions {
-    filter?: string;
-    format?: OdataMetadataFormat;
-    select?: string;
-    top?: number;
-}
+export type QueryOptions = Omit<TableQueryOptions, "select"> & {
```

Comment:
To confirm my understanding of what's going on here: The goal of this PR is to provide a nicer interface when you want to only select a few properties from a query operation.  Instead of having to pass a string with property names separated by commas (which is what the service API expects), we use a `string[]`, since that is easier for end users, and then do the conversion before calling into the generated model?

Reply:
Yes that is correct! I'm updating the PR description